en/xml/fo/topic.graphic-titlepage.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    License CC BY-NC-SA 3.0
+
+   This work is licensed under the
+   "Namensnennung – Keine kommerzielle Nutzung – Weitergabe unter
+    gleichen Bedingungen 3.0 Deutschland (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/de/deed.de
+
+   Read the English translation here:
+
+   "Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/
+
+-->
+<?xml-model href="file:../5.1/dbref.rnc" type="application/relax-ng-compact-syntax"?>
+<section xml:id="dbc.fo.graphic-titlepage" remap="topic" userlevel="easy"
+    xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Graphics on Title Pages</title>
+    <subtitle>Place and format an individual graphic on a title page</subtitle>
+    <info>
+        <definitions definitionfile="defs.xml"/>
+        <keywordset>
+            <keyword>title page</keyword>
+            <keyword>cover</keyword>
+            <keyword>graphic</keyword>
+            <keyword>block</keyword>
+            <keyword>format</keyword>
+        </keywordset>
+        <subjectset>
+            <subject>
+                <subjectterm>graphic</subjectterm>
+            </subject>
+        </subjectset>
+    </info>
+
+    <section role="problem">
+        <title>Problem</title>
+        <para>You need to know how to add a graphic to your titlepage.</para>
+    </section>
+    <section role="solution">
+        <title>Solution</title>
+        <para>To add a graphic to one of your titlepages, you need to creata a custom layer for
+            titlepages as described in the <xref linkend="dbc.fo.design-titlepages"/>section.</para>
```

Comment:
I've found the following issues in the first paragraph:

* Typo: creata -> create
* Avoid combining an `<xref/>` with the word "section", "chapter" etc. When an `<xref/>` is resolved, it'll normally contains already the word section.
* Term: "custom layer" is actually wrong. It is either "customization layer" or probably better use "template".

With all the above issues taken into account, I would rephrase the sentence like this:

```xml
<para>
To add a graphic to one of your titlepages, create a template as described
in <xref linkend="dbc.fo.design-titlepages"/>.
</para>
```